pdns/dnsdist-xpf.cc

```diff
@@ -52,6 +52,7 @@ bool addXPF(DNSQuestion& dq, uint16_t optionCode)
   pos += sizeof(drh);
   memcpy(reinterpret_cast<char*>(&data.at(pos)), payload.data(), payload.size());
   pos += payload.size();
+  (void) pos;
```

Comment:
This is to mark the result from the previous line used, and you don't want to delete the previous line in case we ever add more code?

Reply:
Exactly. I can live with deleting the previous line if you think it's cleaner, though :)

Reply:
What you did makes sense :)